universal_notifications/backends/emails.py

```diff
@@ -31,7 +32,7 @@ def send_email(template, to, subject, variables={}, fail_silently=False, cms=Fal
     replace_variables['protocol'] = protocol
     domain = variables['site'].domain
     replace_variables['domain'] = domain
-    for key, value in replace_variables.iteritems():
+    for key, value in (replace_variables.iteritems() if six.PY2 else replace_variables.items()):
```

Comment:
It will but probably some our projects and users will stick with py2.
I agree with not using `if` but built-in six functions looks pretty and easy to replace when we drop py2 support.